ftests/core/src/main/java/org/commonjava/indy/data/DefaultStoreValidatorTest.java

```diff
@@ -106,21 +106,53 @@ public void run() throws Exception {
             new RemoteRepository( "maven", "validation-indy-allowed", "http://127.0.0.1" );
 
         LOGGER.warn("=> Start Validating RemoteRepository: [" + allowedRemoteRepo.getUrl()+"]");
-        RemoteRepository remoteRepository3 = client.stores().create(allowedRemoteRepo, changelog,
-            RemoteRepository.class);
+        RemoteRepository remoteRepository3 = client.stores().create(allowedRemoteRepo, changelog, RemoteRepository.class);
 
 
         ArtifactStoreValidateData validateAllowedRepo = validator.validate(remoteRepository3);
-        LOGGER.warn("=> Returned [Allowed Not Valid ( !GET | HTTP ) SSL] ArtifactStoreValidateData: " + validateAllowedRepo.toString());
+        LOGGER.warn("=> Returned [Allowed Not Valid ( GET | HTTP ) SSL] ArtifactStoreValidateData: " + validateAllowedRepo.toString());
 
         assertNotNull( validateAllowedRepo );
         assertFalse(validateAllowedRepo.isValid());
-        assertNull(validateAllowedRepo.getErrors().get("HTTP_GET_STATUS"));
-        assertNull(validateAllowedRepo.getErrors().get("HTTP_HEAD_STATUS"));
+        assertNotNull(validateAllowedRepo.getErrors().get("HTTP_GET_STATUS"));
+        assertNotNull(validateAllowedRepo.getErrors().get("HTTP_HEAD_STATUS"));
         assertNotNull( validateAllowedRepo.getErrors().get("disabled") );
 
 
 
+        RemoteRepository npmRemoteRepo =
+          new RemoteRepository( "npm", "validation-indy-npm", "https://repository.engineering.redhat.com/nexus/" );
+
+        LOGGER.warn("=> Start Validating NPM RemoteRepository: [" + npmRemoteRepo.getUrl()+"]");
+        RemoteRepository npmRemoteRepository = client.stores().create(npmRemoteRepo, changelog, RemoteRepository.class);
+
+
+        ArtifactStoreValidateData validateNPMRepo = validator.validate(npmRemoteRepository);
+        LOGGER.warn("=> Returned [Allowed Not Valid ( GET | HTTP ) SSL] ArtifactStoreValidateData: " + validateNPMRepo.toString());
+
+        assertNotNull( validateNPMRepo );
+        // Valid SSL remote repo but because there is authentication HTTP GET & HEAD are unsuccessfull 
+        assertFalse(validateNPMRepo.isValid());
+        assertNotNull(validateNPMRepo.getErrors().get("HTTP_GET_STATUS"));
+        assertNotNull(validateNPMRepo.getErrors().get("HTTP_HEAD_STATUS"));
+
+
+        RemoteRepository testAllowedRemoteRepo =
+          new RemoteRepository( "maven", "validation-indy-test-allowed", "http://maven.repository.redhat.com/techpreview/all" );
```

Comment:
We shouldn't depend on remote servers for our ftests / unit tests if possible, because any number of network / server errors outside out control can cause tests to fail.